public/style.css

```diff
@@ -0,0 +1,44 @@
+body{
+  color: #02075d;
```

Comment:
Good use of a separate stylesheet as opposed to inline CSS which may affect readability of code